README.md

```diff
@@ -0,0 +1,38 @@
+This project contains the source code of the backend of the smart city dashboard. It fetches data from different sources and serves as a relay for the [frontend](https://github.com/reedu-reengineering-education/smart-city-dashboard)
```

Comment:
Punkt am Ende. (Sorry, ich kann irgendwie die Datei nicht direkt für solche Kleinigkeiten ändern).

Reply:
Resolved in [7857af4](https://github.com/reedu-reengineering-education/smart-city-dashboard-backend/pull/5/commits/7857af4c135f4859a599b54fcc223f03c97239a9)